assets/variant-list.js

```diff
@@ -283,12 +300,12 @@ class VariantList extends HTMLElement {
   }
 
   updateLiveRegions(id, message) {
-    const variantItemError = document.getElementById(`Variant-item-list-${id}`);
-    if (variantItemError) variantItemError.querySelector('.variant-list__error-text').innerHTML = message;
+    const variantItemError = document.getElementById(`Variant-list-item-error-${id}`);
+    if (variantItemError) variantItemError.querySelector('.variant-item__error-text').innerHTML = message;
 
     this.variantItemStatusElement.setAttribute('aria-hidden', true);
 
-    const cartStatus = document.getElementById('cart-live-region-text');
+    const cartStatus = document.getElementById('variant-list-live-region-text');
     cartStatus.setAttribute('aria-hidden', false);
 
     setTimeout(() => {
```

Comment:
@sofiamatulis @KaichenWang 

So, with this solution we would be moving away from inline error messaging? 

Do you think this would make for a bad experience on Mobile because the error messaging would be all the way at the bottom of the section (because the footer isn't sticky on Mobile)? 

Buyers would have to scroll down to the see the error message and then scroll back up to the variant they were interacting with.

Sorry, please correct me if I'm wrong on this one!

Reply:
@sofiamatulis my bad! I'm a bit slow this morning, sorry. 🤦‍♂️ 

Okay! This sounds good and all makes sense now. 
Can we match the same error styling with the red icon though? 
https://screenshot.click/Same-error-styling.png

cc: @KaichenWang